syllabus/7-Polkadot/README.md

```diff
@@ -2,22 +2,66 @@
 
 ## Learning Outcomes
 
+* Mission
+    * What are the goals behind Polkadot?
+* Elections
+    * How do we get validators to play our game?
+    * NPoS
+* Consenus
+    * How validators play the consensus game?
+    * Data collation
+    * Availability sharding
+    * Execution sharding
+    * Scheduling (Coretime)
+    * XCMP
+* Governance
+    * How does the system evolve?
+    * OpenGov
+    * Fellowship
+* Tooling
+    * What extra tools are available to build on Polkadot?
+    * Cumulus
+    * Zombienet
+
 ## Hands on activities
 
-## Schedule
+- Build a parachain (maybe)
+- Malicious validator zombienet testing (maybe)
+- Quiz
+
+# Schedule
 
 ## Day 1
 
 ### Morning
 
+- Introduction to Polkadot part 2 [Maciej]
+- Data Preparation - Collation & Backing [Maciej]
+- Data Sharding - Data Availability [Bradley]
+- Execution Sharding - Approvals and Disputes [Maciej]
+
 ### Afternoon
 
+- Coretime [Maciej]
+- Cumulus [Bradley]
+- Asynchronous Backing [Bradley]
+- XCMP [? > Maciej]
+
 ## Day 2
 
 ### Morning
 
+- Light Clients [Nikos]
+- NPoS & Staking [Ankan > Kian > Maciej]
```

Comment:
I have an old lecture on this that still holds, and would be glad to re-lecture it. Ankan will be able to mirror if need be as well, so I you can be confident that we can take care of this if it ends up being in the schedule.